java/statestore/src/main/java/sleeper/statestore/InitialiseStateStore.java

```diff
@@ -126,44 +109,7 @@ public static void main(String[] args) throws StateStoreException, IOException {
             stateStore = new DynamoDBStateStore(tableProperties, dynamoDBClient);
         }
 
-        List<Object> splitPoints = null;
-        if (args.length > 2) {
-            splitPoints = new ArrayList<>();
-            String splitPointsFile = args[2];
-            boolean stringsBase64Encoded = 4 == args.length && Boolean.parseBoolean(args[2]);
-
-            PrimitiveType rowKey1Type = tableProperties.getSchema().getRowKeyTypes().get(0);
-            List<String> lines = new ArrayList<>();
-            try (BufferedReader reader = new BufferedReader(new InputStreamReader(
-                    Files.newInputStream(Paths.get(splitPointsFile)), StandardCharsets.UTF_8))) {
-                String lineFromFile = reader.readLine();
-                while (null != lineFromFile) {
-                    lines.add(lineFromFile);
-                    lineFromFile = reader.readLine();
-                }
-            }
-            for (String line : lines) {
-                if (rowKey1Type instanceof IntType) {
-                    splitPoints.add(Integer.parseInt(line));
-                } else if (rowKey1Type instanceof LongType) {
-                    splitPoints.add(Long.parseLong(line));
-                } else if (rowKey1Type instanceof StringType) {
-                    if (stringsBase64Encoded) {
-                        byte[] encodedString = Base64.decodeBase64(line);
-                        splitPoints.add(new String(encodedString, StandardCharsets.UTF_8));
-                    } else {
-                        splitPoints.add(line);
-                    }
-                } else if (rowKey1Type instanceof ByteArrayType) {
-                    splitPoints.add(Base64.decodeBase64(line));
-                } else {
-                    throw new RuntimeException("Unknown key type " + rowKey1Type);
-                }
-            }
-            LOGGER.info("Read " + splitPoints.size() + " split points from file");
-        }
-
-        new InitialiseStateStore(tableProperties, stateStore, splitPoints).run();
+        InitialiseStateStore.createInitialiseStateStoreFromSplitPoints(tableProperties, stateStore, null).run();
```

Comment:
It might be clearer to use an empty list for the split points rather than null.

Reply:
Done.